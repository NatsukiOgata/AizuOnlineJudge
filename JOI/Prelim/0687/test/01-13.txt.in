100
1258 1642 1065 1618 1236 1235 1497 1597 1603 1941 2000 1701 1416 1290 1195 1184 1815 1674 1905 1836 1855 1012 1668 1003 1054 1921 1029 1595 1226 1592 1819 1000 1714 1883 1876 1880 1114 1785 1984 1694 1929 1149 1374 1130 1110 1598 1648 1527 1393 1591 1624 1040 1571 1343 1297 1175 1753 1622 1791 1631 1189 1246 1265 1423 1633 1008 1954 1548 1599 1407 1238 1804 1717 1605 1371 1520 1453 1327 1103 1373 1938 1768 1188 1428 1686 1076 1552 1415 1563 1912 1198 1944 1163 1328 1611 1192 1092 1967 1691 1335
