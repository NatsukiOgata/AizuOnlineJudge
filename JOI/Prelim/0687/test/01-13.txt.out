14692
132846
